client/src/app/components/page/login-page/login-page.component.scss

```diff
@@ -1,7 +1,9 @@
 .formContainer{
```

Comment:
Remove all style from here. You don't need them. Use Bootstrap elements only. Use custom styling only and only when you really cannot style with UI framework.

Reply:
I deleted all custom scss styles

Reply:
Removed